dosma/core/io/dicom_io.py

```diff
@@ -206,6 +206,7 @@ def load(
         sort_by: Union[str, int, Sequence[Union[str, int]]] = np._NoValue,
         ignore_ext: bool = np._NoValue,
         default_ornt: Tuple[str, str] = np._NoValue,
+        keep_pixel_data: bool = False,
```

Comment:
We probably don't need this parameter. I would remove the pixel data by default - i have a PR for this, will merge it and we can rebase this PR on that one